Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.13-alpine
+
+RUN apk update
+RUN apk add apache2-utils bash
+RUN python -m venv /maildump
+RUN /maildump/bin/pip install maildump
+
+ENV MAILDUMP_PASSWORD=
+
+ADD docker/entrypoint.sh /
+RUN mkdir /maildump/data
+RUN chmod +x /entrypoint.sh
```

Comment:
I think you can skip this (and thus avoid another layer in the image) if you make the original file executable instead.